src/main/java/hw10/FileLogger.java

```diff
@@ -0,0 +1,79 @@
+package hw10;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Objects;
+
+public class FileLogger {
+    private static final DateTimeFormatter fileDateFormatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final FileLoggerConfiguration configuration =
```

Comment:
configuration should not be a constant. Initialize it in the constructor, please.